src/features/mcda/components/PopupMCDA.tsx

```diff
@@ -0,0 +1,48 @@
+import s from './PopupMCDA.module.css';
+import type { JsonMCDA } from '../atoms/mcdaCalculation';
+
+export type PopupMCDAProps = {
+  json: JsonMCDA;
+  normalized: {
+    [key: string]: { norm: number; val: number };
+  };
+  resultMCDA: number;
+};
+
+export const PopupMCDA = ({ json, normalized, resultMCDA }: PopupMCDAProps) => (
```

Comment:
check in with @alesiahil if we have standard popup components planned that can express this

Reply:
we don't have it right now, it's planned for the future

Reply:
From now, we have ready design here https://www.figma.com/file/IBf3mXhCkAcvGmJ0VzXA60/Layers-Features?node-id=5579%3A171026&t=YRH3FvRFhAwtQZP4-1 which is going to be implemented within the task https://kontur.fibery.io/Tasks/Task/Create-field-10---table-14979 after this US https://kontur.fibery.io/Tasks/User_Story/Map-popup-component-1642 is finished.